### DISCUSSION TITLE
Through pitunneling(tunneling) I am unable to get WebRTC in SIM based Router.

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#2192] [Questions]
Out of the curiosity I tested the my cameras connecting to the SIM based router and without using NAT I brought a pitunnel from pitunnel.com.

It was working perfectly even in the situations where we have firewall. I am getting all WebRTC, HLS..,

But, here my problem is through pitunneling I am unable to get WebRTC. I am able to get HLS in the SIM based Router.
In the local I am able to get WebRTC with the link given the pitunnel.com

Can you please check and suggest.
Can you please let me know is there any other alternatives to get cameras feed without NAT.
Is there any other alternatives like pitunnel which I can get feed even from SIM based Routers. Thank you. 

As your the expert in the video streaming some new things I am learning from. Thanks for your Time and efforts

**Comment by aler9** [comment]
Hello, the README explains how to deal with connectivity issues with WebRTC:

https://github.com/bluenviron/mediamtx#connectivity-issues

[a comment by an automated account is not shown]
